backend/grant/admin/views.py

```diff
@@ -566,6 +579,9 @@ def edit_contribution(contribution_id, proposal_id, user_id, status, amount, tx_
     # Transaction ID (no validation)
     if tx_id:
         contribution.tx_id = tx_id
+    # Refund TX ID (no validation)
+    if refund_tx_id:
+        contribution.refund_tx_id = refund_tx_id
```

Comment:
After `tx_id` is set, and almost certainly after `refund_tx_id` is set we will want to at least warn the admin that if they delete `tx_id` it would be bad. 

Or invalidate empty strings for `tx_id` edits to keep it reverting to pending status.

Reply:
I'm just noting this here, I think there are probably other state-gotchas lurking as well. 

Might make the most sense to work these out during the testing/QA phase. We'll have more of a wholeisting mindset then.

Reply:
I made an issue for this that we could hit as many as possible in one go towards the end, ideally allowing us to come up with a consistent UX for this. #240.